package.json

```diff
@@ -5,7 +5,6 @@
     "build": "athloi run build --concurrency 3 && npm run build-storybook",
     "build-only": "athloi run build",
     "jest": "jest -c jest.config.js",
-    "pretest": "npm run build",
```

Comment:
a minor QOL improvement that i can split out if needed. building every component before test is arduous when iterating on a single component. this lets you do e.g. `make build-x-teaser test`